test/parallel/test-cluster-bind-privileged-port.js

```diff
@@ -26,6 +26,9 @@ const common = require('../common');
 if (common.isOSX)
   common.skip('macOS may allow ordinary processes to use any port');
 
+if (common.isIBMi)
+  common.skip('IBM i may allow ordinary processes to use any port');
```

Comment:
nit: "IBMi allows ordinary processes to use any port" -- lets preserve `grep -i ibmi` by not using "IBM i" instead of "IBMi".

Reply:
@sam-github done.